src/components/plots/plot.ts

```diff
@@ -195,6 +193,12 @@ module Plottable {
         var fn = scale ? (d: any, i: number, dataset: Dataset, m: Plots.PlotMetadata) => scale.scale(accessor(d, i, dataset, m)) : accessor;
         h[attr] = fn;
       });
+      var propertyProjectors = this._propertyToProjectors();
```

Comment:
Going by this line, maybe the method should just be called `_propertyProjectors()`?